core/src/autogluon/core/models/greedy_ensemble/ensemble_selection.py

```diff
@@ -110,19 +110,12 @@ def _fit(self, predictions: List[np.ndarray], labels: np.ndarray, time_limit=Non
         round_scores = False
```

Comment:
The lint check is currently failing, can you please run `black` on this PR? https://github.com/autogluon/autogluon/actions/runs/10262339335/job/28391811858?pr=4367

Reply:
Should also be fixed now.